powerbi-docs/connect-data/desktop-connect-oracle-database.md

```diff
@@ -34,6 +34,8 @@ To determine which version of Power BI Desktop is installed, select **File** > *
 - For the 64-bit version of Power BI Desktop, [download and install the 64-bit Oracle client](https://www.oracle.com/database/technologies/odac-downloads.html).
 
 > [!NOTE]
+> Ensure you choose a version of Oracle Data Access Client (ODAC) which is compatible with your Oracle Server. For instance, ODAC 12.x does not support Oracle Server version 9.
+> It is highly recommended that you choose the Windows installer of the Oracle Client, rather than the xcopy installer.
```

Comment:
I believe they cannot use xcopy installer or they need to do a lot of configuration change.  We don't know exactly what config they need to change if xcopy installer ever works.

Reply:
changed to "Choose the Windows Installer"